### DISCUSSION TITLE
Unable to Retrieve Projects (beta) Node ID

### DISCUSSION THREAD

**Discussion by bpmooch** [community/community#15509] [Projects and Issues]
I am unsure on the proper way to retrieve the node id for my team's project. We are attempting to pull some metrics into a report. Currently, I am trying to retrieve the node id using the REST API, specifically `orgs/{org}/projects` but looking through discussion here, that does not seem to be the right way. Using the GraphQL API, how do I query my organization for Projects (beta)?

**Comment by Martin-Fiser** [comment]
the same problem, perhaps switching back to JIRA

**Comment by bpmooch** [comment]
I never figured out a solution to this. I ended up not tracking the information using projects because I couldn't reliably pull the information out of the API. **big bummo**
